source/assets/css/_imports/dev-guide.less

```diff
@@ -0,0 +1,342 @@
+.start-guide {
+    .dev-guide {
+        text-align: center;
+
+        img {
+            margin-top: 0;
+            width: 120px;
+        }
+    }
+
+    h1:first-of-type {
+        color: #142432;
+        font-size: 33px;
+        font-weight: 600;
+        margin-bottom: 0;
+        padding-bottom: 0;
+    }
+
+    .info.guide {
+        font-weight: 600;
+    }
+
+    .anchorjs-icon:before {
+        content: '';
+    }
+}
+
+/*------- START & END COLUMN -------*/
+.dev-guide--opener,
+.dev-guide--gitter {
```

Comment:
Same here, two classes which are doing the same stuff. Create a new class and apply it on both elements

Reply:
use now: guide--container